kbs/src/api/src/token/coco.rs

```diff
@@ -41,12 +68,18 @@ impl AttestationTokenVerifier for CoCoAttestationTokenVerifier {
 
         let jwk_value = claims_value["jwk"].as_object().ok_or_else(|| anyhow!("CoCo Attestation Token Claims must contain public key (JWK format) to verify signature"))?;
         let jwk = serde_json::to_string(&jwk_value)?;
+        let rsa_jwk = serde_json::from_str::<RsaJWK>(&jwk)?;
```

Comment:
Do we have a unit test that can read a RsaJWK that exported from CoCoAS? The definitions are a little bit different https://github.com/confidential-containers/kbs/pull/265/files#diff-0c923b04d229848729ac1682c38c345239a277ef23fc743c92d3f7e1ba9e675fR113-R120 https://github.com/confidential-containers/kbs/pull/265/files#diff-cdb725680d5fff6ed34b3eea443b0ea1db1135e30aecd9e1eb20b1ec80aee2a7L64-L69

Reply:
Good suggestion, I'll add this unit test.